integrationTests/cypress/support/step_definitions/legal_urls.js

```diff
@@ -9,3 +9,16 @@ Then(/^a footer with three legal links is displayed$/, function () {
     cy.get('footer').get('[data-testId=footer-link-legal-url-imprint]').contains('Imprint').should('have.attr', 'href', Cypress.env("ImprintURL"))
     cy.get('footer').get('[data-testId=footer-link-privacy-policy]').contains('Privacy Policy').should('have.attr', 'href', Cypress.env("PrivacyPolicyURL"))
 });
+
+
+Then(/^the legal links in the footer are translated to german$/, function () {
```

Comment:
You can simply use `"` instead of `/^` and `$/`, i.e. "the legal links in the footer are translated to german"

Please also note the other steps that have been added.